resources/widgetlist.js

```diff
@@ -114,12 +114,13 @@ class WidgetList extends HTMLElement {
     // reach the next row/column.
     this._yinc2 = 1;
 
-    if (this.scale != null) this._scale = Math.max(parseFloat(this.scale), 0.01);
-    if (this.elementwidth != null) this._elementwidth = Math.max(parseInt(this.elementwidth), 1);
-    if (this.elementheight != null) this._elementheight = Math.max(parseInt(this.elementheight), 1);
-    if (this.rowcolsize != null) this._rowcolsize = Math.max(parseInt(this.rowcolsize), 1);
-    if (this.toward != null) this.parseToward(this.toward);
-    if (this.maxnumber != null) this._maxnumber = Math.max(parseInt(this.maxnumber), 1);
+    if (this.scale !== null) this._scale = Math.max(parseFloat(this.scale), 0.01);
+    if (this.elementwidth !== null) this._elementwidth = Math.max(parseInt(this.elementwidth), 1);
+    if (this.elementheight !== null)
+      this._elementheight = Math.max(parseInt(this.elementheight), 1);
```

Comment:
This exceeded the maximum line length 😭

Reply:
Is it reasonable to convert all `if-null-checking` to bool operators?

for example
```javascript
this._elementheight = Math.max(parseInt(this.elementheight || "1"), 1);
```

Reply:
Since there's default values above, we could also do plain ol' ternaries:
```js
this._elementheight = this.elementheight ? Math.max(parseInt(this.elementheight), 1) : 100;
```
Which doesn't add a lot, but it does bring the line length down slightly.

Reply:
Using a boolean or a ternary is not possible, because it conflates falsy values like `0` or `false` with `null`.  In this case, setting to zero is not the same as leaving it null (and using the default value).   A util function is fine, although I think that could be done as a followup.

Reply:
```suggestion
    this._elementheight = Math.max(parseInt(this.elementheight ?? 100), 1);
```

Reply:
Unfortunately, OverlayPlugin is not using a recent enough version of Chrome to use nullish coalescing operators.

Reply:
Babel after Webpack lands? ;)